frontend/public/components/modals/cluster-update-modal.tsx

```diff
@@ -126,41 +141,93 @@ const ClusterUpdateModal = withHandlePromise((props: ClusterUpdateModalProps) =>
       close,
     );
   };
-  const options = availableSortedUpdates.map(({ version }) => {
-    return (
-      <SelectOption
-        key={version}
-        value={version}
-        isDisabled={clusterUpgradeableFalse && isMinorVersionNewer(currentVersion, version)}
-      />
-    );
+  const dropdownItem = (version) => {
+    const isDisabled = clusterUpgradeableFalse && isMinorVersionNewer(currentVersion, version);
+    return {
+      isDisabled,
+      key: version,
+      title: isDisabled ? (
+        <>
+          {version} <UpdateBlockedLabel />
```

Comment:
`Update blocked` (which seems to be the current string behind `UpdateBlockedLabel`) is a bit strong.  These targets are not recommended, but they are still supported, and admins should be able to select them if they have enabled the `Include supported but not recommended versions` toggle.